assets/base.css

```diff
@@ -1585,6 +1585,7 @@ details[open] > .share-button__fallback {
   appearance: none;
   background-color: rgb(var(--color-background));
   color: rgb(var(--color-foreground));
+  font-family: var(--font-body-family);
```

Comment:
I think here what Kai mentioned is still missing. For example we can see the different on the contact page/form: 
<details>

<summary>Screenshot</summary>

![](https://screenshot.click/13-24-z5u6h-nyz6a.png)

</details>